pool.go

```diff
@@ -123,7 +123,7 @@ func NewPool(size int, options ...Option) (*Pool, error) {
 		}
 	}
 	if p.options.PreAlloc {
-		if size == -1 {
+		if size < 0 {
```

Comment:
`size` has been re-assigned to -1 in the entry of `NewPool`: https://github.com/panjf2000/ants/blob/master/pool.go#L100-L102